src/main/java/org/jitsi/jigasi/JvbConference.java

```diff
@@ -1201,20 +1212,9 @@ private void processChatRoomMemberLeft(ChatRoomMember member)
                         return false;
                     }
 
-                    try
-                    {
-                        String jidString = roomName  + "/" + m.getName();
-                        DiscoverInfo i = ServiceDiscoveryManager.getInstanceFor(getConnection())
-                            .discoverInfo(JidCreate.entityFullFrom(jidString));
-
-                        if (!i.containsFeature(SIP_GATEWAY_FEATURE_NAME))
-                        {
-                            return true;
-                        }
-                    }
-                    catch(Exception e)
+                    if (!jigasiChatRoomMembers.contains(m.getName()))
```

Comment:
`return !jigasiChatRoomMembers.contains(m.getName())`?

I find all this hard to read, I'd do something like 
```
onlyJigasisInRoom = allMatch(m ->
  m.getName().equals(getResourceIdentifier().toString() 
  || m.getName().equals(gatewaySession.getFocusResourceAddr()) 
  || jigasiChatRoomMembers.contains(m.getName()));
 ```

Reply:
There's also `ChatRoomMemberJabberImpl#getLastPresence()` that you can use to make a `isJigasi` utility an avoid saving jigasi names in a list.

Reply:
There was something about it that prevents me from using it ... I need to double check

Reply:
I cannot use getLastPresence(), cause jitsi xmpp PP does not store the first presence ... only the updates after creating the member. I will leave it like this, as it needs more refactoring in jitsi-desktop.